File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/Appointments/Appointment.cs
using System;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models.Appointments
{
    [Serializable]
    public class Appointment
    {
        [XmlAttribute("apptId")]
        public string ApptId { get; set; }

        [XmlAttribute("startDate")]
        public string StartDate { get; set; }

        [XmlAttribute("endDate")]
        public string EndDate { get; set; }

        [XmlAttribute("details")]
        public string Details { get; set; }

        [XmlAttribute("siteName")]
        public string SiteName { get; set; }

        [XmlAttribute("address")]
        public string Address { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/Appointment.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "Appointment")
@XmlAccessorType(XmlAccessType.FIELD)
data class Appointment(
    @XmlAttribute var apptId: String = "default apptId",
    @XmlAttribute var startDate: String = "default startDate",
    @XmlAttribute var endDate: String = "default endDate",
    @XmlAttribute var details: String = "default details",
    @XmlAttribute var siteName: String = "default siteName",
    @XmlAttribute var address: String = "default address"
)
